container-host-files/etc/scf/config/opinions.yml

```diff
@@ -452,7 +449,7 @@ properties:
       # go into the SCF zone.
       app_autoscaler:
         # Requires read-only admin to look up stats, and write admin to do scaling
-        authorities: cloud_controller.read,cloud_controller.write,cloud_controller.admin
+        authorities: cloud_controller.read,cloud_controller.write,cloud_controller.admin,uaa.resource
```

Comment:
I don't know what the `uaa.resource` scope does; why does the autoscaler need it?

Reply:
this is due to a functional change in app-autoscaler in which we need to allow cf admin's access to autoscaler resourc.  To validate admin role,  the  uaa.resource authority is required to invoke /token endpoint.